keras_core/ops/math.py

```diff
@@ -929,3 +929,37 @@ def rsqrt(x):
         return Rsqrt().symbolic_call(x)
     x = backend.convert_to_tensor(x)
     return backend.math.rsqrt(x)
+
+
+class Erf(Operation):
+    def compute_output_spec(self, x):
+        return KerasTensor(shape=x.shape, dtype=x.dtype)
+
+    def call(self, x):
+        return backend.erf(x)
+
+
+@keras_core_export("keras_core.ops.erf")
+def erf(x):
+    """Computes the error function of x element-wise.
+
+    Args:
+        x: input tensor
+
+    Returns:
+        A tensor with the same type as `x`.
+
+    Examples:
+
+    # Basic usage
```

Comment:
Since you're not printing any outputs, just use a fenced code block for the code example.

Reply:
Done